src/views/FAQPage/index.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="view-container">
+    <div class="data-list faq-data-list">
+      <div
+        v-for="(item, index) in dataItems"
+        :key="index"
+        class="data-item faq-data-item initBorder"
+        :class="{'isClosed': !item.isOpen}"
+      >
+        <FAQ-Item
+          v-bind:question="item.question"
+          v-bind:answer="item.answer"
+          v-bind:toggleAccordion="toggleAccordion"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import fetch from 'node-fetch';
+// eslint-disable-next-line import/extensions
```

Comment:
this shouldnt be throwing a linting error, please check your config files

Reply:
That's strange. I just checked again. It's still throwing me that error. Is there a way to check where this rule is applied from?

Reply:
Is that error saying that you should have the .vue extension in the import? We shouldn't ignore any rule and rather fix them.

Reply:
Yes, that's the lint rule. Should we use .vue extension always while importing?

Reply:
Yup!

Reply:
Alright, cool. Thanks. 
This is another thing I don't like about Vue now. But it is what it is. :p 
Is there a specific reason for this? I'd love to know why this is important.

Reply:
thats actually due to the airbnb eslint config, vue doesnt care either way. but yeah

Reply:
Ah, that's right. But why did Airbnb prefer it that way? I'd think it's cleaner without the extensions.